Configuration.props

```diff
@@ -39,6 +37,8 @@
     <AndroidLatestUnstableApiLevel Condition="'$(AndroidLatestUnstableApiLevel)' == ''">32</AndroidLatestUnstableApiLevel>
     <AndroidLatestUnstablePlatformId Condition="'$(AndroidLatestUnstablePlatformId)' == ''">32</AndroidLatestUnstablePlatformId>
     <AndroidLatestUnstableFrameworkVersion Condition="'$(AndroidLatestUnstableFrameworkVersion)'==''">v12.1</AndroidLatestUnstableFrameworkVersion>
+    <!-- The default API level used for $(TargetPlatformVersion) in .NET 6+ -->
+    <AndroidDefaultTargetDotnetApiLevel Condition=" '$(AndroidDefaultTargetDotnetApiLevel)' == '' ">$(AndroidLatestStableApiLevel)</AndroidDefaultTargetDotnetApiLevel>
```

Comment:
Not sure if/how this should interact with template packs, a'la 6eb11f159c692563200ffb88cddc7b69f761cace & https://github.com/xamarin/xamarin-android/blob/c80dfff7a3183e21d356d2c5835aa0821fd9bd90/src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json#L32-L35

Additionally, this feels potentially "brittle"; "API-33" (currently API-Tiramisu) will *eventually* become the `$(AndroidLatestStableApiLevel)` value, *probably* in the July-August timeframe.  Do we actually want `$(AndroidDefaultTargetDotnetApiLevel)` to update to API-33 at that time, automatically?

Reply:
We can probably drop the 31 pack from the workload. It should keep resolving from NuGet.org if we keep this in place:

https://github.com/xamarin/xamarin-android/blob/25e0eefbd20bb4778a4717e4ef587a080c82bca0/build-tools/create-packs/Microsoft.Android.Sdk.proj#L125-L128

There are usages of `net6.0-android3` on Github:

https://github.com/search?q=net6.0-android31+extension%3Acsproj&type=Code&ref=advsearch